pkgs/development/python-modules/yapsy/default.nix

```diff
@@ -4,9 +4,10 @@
   fetchFromGitHub,
   setuptools,
   pytestCheckHook,
+  python,
 }:
 
-buildPythonPackage {
+with python.pkgs; buildPythonPackage {
```

Comment:
can you explain the reason for this change ?

Reply:
It's `packaging` dependency needs it; In an upcoming commit I've removed the `with python.pkgs` and instead used `python.pkgs.packaging` as the dependency. Thanks.